src/test/java/racingGameTest/RacingGameTest.java

```diff
@@ -1,19 +1,33 @@
 package racingGameTest;
```

Comment:
테스트 클래스는 테스트 대상 클래스와 같은 패키지에 위치하면 어떨까요?

```suggestion
package racingGame.domain;
```